portfolio/src/main/webapp/script.js

```diff
@@ -17,9 +17,59 @@
  */
 
 // Spotify API keys to be used for later
-// var client_id = '444dda318d8949cb9282d95a9885a7c6';
-// var client_secret = '9f81ba6cf5748ef9d24a7ddb1e8bc4c';
-// var redirect_uri = 'https://8080-dot-12317386-dot-devshell.appspot.com/misc.html';
+const clientId = "444dda318d8949cb9282d95a9885a7c6";
+let oauthToken = 'BQCebg7W1ZsfuUCdCjdcHz-GDJFMj5GjOhTxvrpNFhu3gvEzYftF3UrSIZ9KHYXVjVdF346qrcaQtd4df7FN0p5sQj2x2p3HpEsCtIDPg4owl4aZx5HfQ4wqWKY3y8l2KakUPS2ZwGKW_PbiPefFMJ3y2FVGfzmz2Kb--w9DxyLBOA';
+
+
+function minutesAndSeconds(milliseconds) {
```

Comment:
nit: Make this function name more descriptive of what it's actually doing, for example `formatAsMinutesAndSeconds`